app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,44 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name')
+    const postTitle = this.postTitleMarkup;
+
+    // need this data !!
+    const isOwnWall = firstReplierName === '// check model.uri for username? in Message Wall uri';
+    const postRemoved = '// need check to see if post has been removed';
```

Comment:
Something I may have neglected to mention is that each of these mixins corresponds to a single notification type, and those types are listed here https://github.com/Wikia/pandora/blob/master/service/on-site-notifications/src/main/java/com/wikia/services/onsitenotifications/domain/notification/NotificationFactory.java#L32-L34
Each notification's `model` should have a `type` property corresponding to one in that list. That means "post removed" should have its own file handling the formatting for the `wall-post-removed-notification` so there's no trick to determining if the post was removed, that notification type will be dealt with separately.